src/main/java/transforms/ParseBoolTransform.java

```diff
@@ -19,7 +19,7 @@ public ParseBoolTransform() {
     private boolean convertToBoolean( String value ) {
         boolean returnValue = false;
         if ( "1".equalsIgnoreCase( value ) || "yes".equalsIgnoreCase( value ) ||
-                "true".equalsIgnoreCase( value ) || "on".equalsIgnoreCase( value ) ) {
+                "true".equalsIgnoreCase( value ) || "on".equalsIgnoreCase( value ) || "y".equalsIgnoreCase( value ) ) {
```

Comment:
is "on" used to mean true more often than being a typo of "no"?

Reply:
I think this is for on/off.

Reply:
That's what I figured -- it just seems like someone writing "on" to mean `true` would happen less often than someone trying to write "no" and making a typo and writing "on"